terraform/deployments/govuk-publishing-platform/signon_secrets.tf

```diff
@@ -66,109 +32,127 @@ locals {
     router_api          = "Router API"
   }
   signon_api_user = {
-    content_store       = join("-", compact([local.api_user_prefix, "content-store@alphagov.co.uk"]))
-    draft_content_store = join("-", compact([local.api_user_prefix, "draft-content-store@alphagov.co.uk"]))
-    frontend            = join("-", compact([local.api_user_prefix, "frontend@alphagov.co.uk"]))
-    publisher           = join("-", compact([local.api_user_prefix, "publisher@alphagov.co.uk"]))
-    publishing_api      = join("-", compact([local.api_user_prefix, "publishing-api@alphagov.co.uk"]))
+    content_store       = join("-", compact([local.api_user_prefix, "content-store@digital.cabinet-office.gov.uk"]))
```

Comment:
Are we sure we want these to share a namespace with staff usernames and anything else that might need a Google Workspace account (including in the future)? Bear in mind that GDS IT owns this namespace, not GOV.UK, so we're not necessarily in control of future things which might want an @digital.cabinet-office.gov.uk address. (Sorry for commenting late - hadn't seen this in the earlier PR where it was first introduced.)

Reply:
Sure, these can be any unique email address. Do you think `<workspace>.<env>.govuk.digital` would be better?

Reply:
Yeah, that sounds like a better bet to me. We (GOV.UK) own the domain and we don't run email on it so much less potential for clashes or confusion 👍

Reply:
Great! @sengi I've pushed this corresponding fix up, lmk if you don't feel comfortable reviewing as it's a Ruby commit https://github.com/alphagov/signon/pull/1631